.vscode/settings.json

```diff
@@ -16,11 +16,8 @@
         "editor.defaultFormatter": "esbenp.prettier-vscode"
     },
     "[java]": {
-    "spotlessGradle.format.enable": true,
-    "editor.defaultFormatter": "richardwillis.vscode-spotless-gradle",
-    "editor.codeActionsOnSave": {
-      "source.fixAll.spotlessGradle": true
-    },
+        "spotlessGradle.format.enable": true,
+        "editor.defaultFormatter": "redhat.java",
```

Comment:
not going to work since this is a different formatter than the one we're relying on (spotless). It might be better to switch formatters altogether. I think I looked into prettier for Java but I don't remember if that was supported or not.

There is a way to do this using GitHub Actions, but I'd rather have formatting fixed pre-commit.

Reply:
I'll look into using prettier today